### PR TITLE
Initial design of the peers application

### PR DESCRIPTION
**Pull request by daminisatya**

### CODE REVIEW

File: html/apps/oneClickDeploy/app.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+var app = angular.module('PeersCount', []);
```
Comment (sudheesh001) [inline html/apps/oneClickDeploy/app.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2620031428/issues/source?bid=3288751&fileBranchId=3313965#l1)

File: html/apps/oneClickDeploy/app.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+app.controller('getPeersCount', ['$scope', '$http', '$sce', function($scope, $http, $sce) {
+    $http.jsonp(String('http://loklak.org/api/peers.json')).success(function(response) {
```
Comment (sudheesh001) [inline html/apps/oneClickDeploy/app.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2620031428/issues/source?bid=3288751&fileBranchId=3313965#l3)

File: html/apps/oneClickDeploy/app.js (hunk 1 above)
Comment (sudheesh001) [inline html/apps/oneClickDeploy/app.js]: Codacy Issue found: ["angular" is not defined.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2620031428/issues/source?bid=3288751&fileBranchId=3313965#l1)

File: html/apps/oneClickDeploy/app.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+var app = angular.module('PeersCount', []);
+app.controller('getPeersCount', ['$scope', '$http', '$sce', function($scope, $http, $sce) {
```
Comment (sudheesh001) [inline html/apps/oneClickDeploy/app.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2620031428/issues/source?bid=3288751&fileBranchId=3313965#l2)

**Comment by daminisatya** [comment]
#398

**Comment by daminisatya** [comment]
Merging the initial framework!

File: html/apps/oneClickDeploy/app.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+var app = angular.module("PeersCount", []);
```
Comment (sudheesh001) [inline html/apps/oneClickDeploy/app.js]: Codacy Issue found: ["angular" is not defined.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2620055748/issues/source?bid=3288751&fileBranchId=3313965#l1)

**Pull request merged**